packages/mermaid/package.json

```diff
@@ -59,7 +59,6 @@
     "elkjs": "^0.8.2",
     "khroma": "^2.0.0",
     "lodash-es": "^4.17.21",
-    "moment-mini": "^2.24.0",
```

Comment:
You'll need to add 'moment' here. It might be working now because some other package has dependency on it. But it'll break in the future. 

Also, can you check what the output sizes of mermaid is after this change?

Reply:
moment was already there.. line 88

![image](https://user-images.githubusercontent.com/12377432/215243038-9cafd837-fdd3-4e91-b6ae-e2c882124ee1.png)

![image](https://user-images.githubusercontent.com/12377432/215243053-6907ae48-7469-43a3-947e-438aea163789.png)

Reply:
Can you share output of `ls -lh packages/mermaid/dist`

Reply:
![image](https://user-images.githubusercontent.com/12377432/215243187-6fbde73b-e228-4425-ae12-fd85e8b02db2.png)

Reply:
Looks like size has went up. Try `pnpm build:viz` ?

Reply:
![image](https://user-images.githubusercontent.com/12377432/215243386-554afcc8-e593-4f4e-a943-e1f79b44af7a.png)

Reply:
The vizualizasions will be in `stats` folder.

Reply:
-rw-r--r-- 1 emers 197610 Jan 28 03:25 mermaid.js 4857kB => 4901kB (+44KB)
-rw-r--r-- 1 emers 197610 Jan 28 03:25 mermaid.min.js 2307kB => 2308kB (+1KB)
-rw-r--r-- 1 emers 197610 Jan 28 03:25 mermaid.c39b89c7.js 1594kB => 1632kB (+38KB)
-rw-r--r-- 1 emers 197610 Jan 28 03:24 mermaid.beb70bac.js 1115kB => 1236kB (+116KB)
-rw-r--r-- 1 emers 197610 Jan 28 03:24 mermaid.core.mjs 1kB => 1kB
-rw-r--r-- 1 emers 197610 Jan 28 03:25 mermaid.esm.min.mjs 1kB => 1kB
-rw-r--r-- 1 emers 197610 Jan 28 03:25 mermaid.esm.mjs 1kB => 1kB

Reply:
> moment was already there.. line 88
> 

But that's under devDependency. We need moment as a dependency.

Reply:
> But that's under devDependency. We need moment as a dependency.

Fixed by https://github.com/mermaid-js/mermaid/pull/4034/commits/df10d649896a660e3b81a7584069d633918401c9, thanks @emersonbottero!